code/modules/clothing/head/helmet.dm

```diff
@@ -216,10 +216,10 @@
 	icon_state = "helm_heavy"
 	item_state = "helm_heavy"
 	armor = list(
-		melee = ARMOR_MELEE_VERY_HIGH,
-		bullet = ARMOR_BALLISTIC_RIFLE,
+		melee = ARMOR_MELEE_MAJOR,
+		bullet = ARMOR_BALLISTIC_MAJOR,
 		laser = ARMOR_LASER_MEDIUM,
-		energy = ARMOR_ENERGY_RESISTANT,
+		energy = ARMOR_ENERGY_SMALL,
```

Comment:
If you want to change the merc armor, do it in another PR, first to atomize it and second because it goes directly to maintainer discussion since they don't need any less survivability, quite the contrary

Reply:
i dont see why it would need a maintainer discussion since this would bring it inline with the Merc Heavy armour vests current values. Instead of randomly having ballistic and riot armour level protection. With the new values, merc armour is no less stronger than sec heavy armour, just consistent.

Reply:
Because I don't agree with it being reduced, as I said they don't need lower survivability, quite the contrary

Reply:
I dont understand why the merc helmet needs such high ballistic and melee stats (equivalent to sec ballistic and riot armour) while the plate carrier uses the standard for heavy armour stats. Its an inconsistency among all our current heavy armours, this is just correcting it. Practically this doesent change their overall survivability too, since their heavy armour plates are unchanged. Again this is for the sake of consistency with our armour sets, i still do not see the issue.

Reply:
We can buff the plate carrier if need be, they however don't have an easy way to heal their brain (no akysine or much of medical support) so it makes sense to protect it more. Either way, this needs to be splitted in a different PR if you want to propose it.

Reply:
i can split it into another PR, but i disagree on buffing the plate carrier. Mercs have Sanna to heal and high DPS weapons to keep security at bay. Buffing merc armour and only merc armour also goes agains the consistency in armour types we currently have, with light - > heavy - > specialist type armour.